bgpd/bgp_nb_config.c

```diff
@@ -923,15 +927,15 @@ int bgp_global_route_selection_options_multi_path_as_set_modify(
 		return NB_OK;
 	case NB_EV_APPLY:
 		bgp = nb_running_get_entry(args->dnode, NULL, true);
+		set = yang_dnode_get_bool(args->dnode, NULL);
 
-		if (!CHECK_FLAG(bgp->flags, BGP_FLAG_MULTIPATH_RELAX_AS_SET)) {
+		if (set
```

Comment:
I think we don't need to check the flag before setting it?

Reply:
true, I will remove the checks